extensions/spring-cloud-config-client/runtime/src/main/java/io/quarkus/spring/cloud/config/client/runtime/SpringCloudConfigServerClientConfig.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.spring.cloud.config.client.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BOOTSTRAP, name = SpringCloudConfigServerClientConfig.NAME)
+public class SpringCloudConfigServerClientConfig {
+
+    protected static final String NAME = "sccsc";
+    /**
+     * If enabled, configuration will be attempted to be read from a Spring Cloud Config Server
```

Comment:
```suggestion
     * If enabled, will try to read the configuration from a Spring Cloud Config Server
```
?

Reply:
Went with your suggestion since it does sounds better than what I had